core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -364,6 +364,20 @@ package object config {
       .checkValues(Set("keytab", "ccache"))
       .createWithDefault("keytab")
 
+  private[spark] val YARN_FILESYSTEMS_TO_ACCESS =
```

Comment:
See other comment. No need to keep this.

Reply:
Removed.